SYCL/Reduction/reduction_range_2d_rw.cpp

```diff
@@ -7,8 +7,8 @@
 // RUNx: %ACC_RUN_PLACEHOLDER %t.out
 
 // This test performs basic checks of parallel_for(range<2>, reduction, func)
-// with reductions initialized with 1-dimensional discard_write accessor
-// accessing 1 element buffer.
+// with reductions initialized with a one element buffer and
+// an initialize_to_identity property.
```

Comment:
I think this comment was supposed to be used for reduction_*dw tests, not *rw tests, right?

Reply:
Somehow mixed the rw/dw files. Should be fixed now, thanks!